spark/src/test/scala/org/apache/spark/sql/delta/skipping/ClusteredTableTestUtils.scala

```diff
@@ -207,15 +207,26 @@ trait ClusteredTableTestUtilsBase extends SparkFunSuite with SharedSparkSession
       s"$locationClause AS SELECT * FROM $srcTable")
   }
 
+  def verifyClusteringColumnsWithoutHistory(
```

Comment:
Note: can't use default parameters on `verifyClusteringColumn` because the method is overloaded.

Reply:
We should update `verifyDescribeHistoryOperationParameters` instead of skip the verification.